lib/App/index.js

```diff
@@ -3,35 +3,42 @@
 
 'use strict';
 
-const querystring = require('querystring');
+const {
+  join,
+  extname,
+  basename,
+  dirname,
+} = require('path');
```

Comment:
Please use `path.join()` in code, otherwise it's hard to trace where a function comes from.